HunterPie.Core/Game/IGame.cs

```diff
@@ -19,9 +19,12 @@ public interface IGame : IDisposable
 
     public int Deaths { get; }
 
+    public int QuestLife { get; }
+
     public event EventHandler<IMonster> OnMonsterSpawn;
     public event EventHandler<IMonster> OnMonsterDespawn;
     public event EventHandler<IGame> OnHudStateChange;
     public event EventHandler<TimeElapsedChangeEventArgs> OnTimeElapsedChange;
     public event EventHandler<IGame> OnDeathCountChange;
+    public event EventHandler<IGame> OnQuestLifeChange;
```

Comment:
There's no need for an event for `Max_____` properties. If you look at every class that implements something like `Value` and `MaxValue`, only the `Value` has an event, because having an event for `MaxValue` is redundant and unnecessary.